app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -189,14 +189,14 @@
           </h3>
         {{/if}}
       </div>
-      <button type="button" class="ui blue small button" {{action 'addTicket' 'free' data.event.tickets.length}}>
+      <button type="button" class="ui blue {{if device.isMobile 'field fluid'}} small button" {{action 'addTicket' 'free' data.event.tickets.length}}>
```

Comment:
I think ~field fluid~ fluid should work. Else its LGTM 👍

Reply:
without field there is no margin between the buttons .
![Screenshot from 2019-11-15 15-43-00](https://user-images.githubusercontent.com/46647141/68935447-aa443980-07be-11ea-8a29-f0709d4ad389.png)